### DISCUSSION TITLE
Outdated elastic integration

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#11917] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

512GB

### Storage for /

15TB

### Storage for /nsm

30TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hi,

how can I force a elastic integration upgrade? Sophos UTM integration is broken at ver. 3.4.0. Is fixed in 3.6.1 and the latest is 3.8.0.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by robbiemarshall** [comment, marked as answer]
The Sophos fix was not included until version 8.11 of Elastic.  The current version of Elastic run by SO 2.4.30 is 8.10.4.  The Sophos fix will be applied when SO upgrades to Elastic 8.11.

**Comment by ByteAndBits** [comment]
Do you have an idea when the upgrade happen? If longer we need to create a workaround for that point. We need this in our datacenter.

**Reply by robbiemarshall** [reply]
Not at this time.  You can follow our blog at https://blog.securityonion.net/ for news about current and upcoming release dates.
